tests/integration/test_kfp_functional.py

```diff
@@ -101,14 +102,15 @@ def assert_get_kfp_deployment(kfp_component: str):
         assert kfp_deployment.status.availableReplicas == 1
 
     for component in KFP_CHARMS:
-        if component == "kfp-api":
+        if component == "kfp-api" or component == "kfp-persistence":
```

Comment:
Note that all this is getting removed in https://github.com/canonical/kfp-operators/pull/282 as it was part of a workaround.

Reply:
Will re-test.